ledger-core/ledger/server/lmnapp/lmn_app_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package lmnapp
+
+import (
+	"testing"
+	"time"
+
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/insconveyor"
+)
+
+func TestStartCtm(t *testing.T) {
```

Comment:
What does this test check?

Reply:
"start compartment"
it runs initial part of LNM compartment. It is skipped, as LMN init logic of SMs isn't ready.